web-api/src/database-types.ts

```diff
@@ -0,0 +1,49 @@
+import { ColumnType, Insertable, Selectable, Updateable } from 'kysely';
+
+export interface Database {
+  message: MessageTable;
+  case: CaseTable;
+}
+
+export interface MessageTable {
+  attachments?: ColumnType<{ documentId: string }[], string, string>;
+  caseStatus: string;
+  caseTitle: string;
+  completedAt?: string;
+  completedBy?: string;
+  completedBySection?: string;
+  completedByUserId?: string;
+  completedMessage?: string;
+  createdAt: string;
```

Comment:
Is there a reason this isn't some sort of timestamp?

Reply:
Will look into this.